config.schema.json

```diff
@@ -28,6 +28,23 @@
               "type": "string",
               "required": true
             },
+            "ignitionOnDuration": {
```

Comment:
We will always need to add any new configuration option to this file. This is what Homebridge uses to create the UI.